### PR TITLE
Refactor tests and support multi-process to speedup

### PR DESCRIPTION
**Pull request by zhangjihang-BD**

### CODE REVIEW

File: tests/accuracy/conftest.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import pytest
+import torch
+
+from tests.utils import get_platform
+
+
+@pytest.fixture(scope="session", autouse=True)
+def setup_session_device(request):
+    platform = get_platform()
+    torch.set_default_device(platform)
```
Comment (LeoLau94) [inline tests/accuracy/conftest.py]: 这个我建议可以弄成另一个fixture，但必须显式调用（也就是不放在conftest里），由于torch全局设置是进程内共享的，所以可以返回一个类对象，__init__时set __del__恢复默认设备
